src/main/java/bio/terra/service/snapshotbuilder/utils/SearchConceptsQueryBuilder.java

```diff
@@ -78,6 +79,7 @@ public Query buildSearchConceptsQuery(
         List.of(
```

Comment:
do we want to update the comment here to include concept_code?

Reply:
I was thinking here that if we plan to remove comments soon, I wouldn't bother trying to keep these up to date. I could remove it instead of not updating if that seems better? Or it can wait for the upcoming ticket which would fix this.

Reply:
I know there is a ticket in the backlog, but because we don't know when that will be done. It's my opinion that we should try to keep things in a good state as much as we can until then (via tests or comments). This will make it easier to do that refactor when that time comes. But, thats just my thought, not a blocker or anything